android/library/src/main/java/com/liferay/mobile/screens/context/SessionContext.java

```diff
@@ -71,7 +72,11 @@ public static Session createSessionFromCurrentSession() {
 	}
 
 	public static boolean isLoggedIn() {
-		return currentUserSession != null;
+		if (currentUserSession == null) {
+			LiferayLogger.e("You need to be logged to view the content!");
```

Comment:
Why showing an error in the console? This method just checks whether there's a session, but it's not an error not having a session